### ISSUE TITLE
Docker Portability

### ISSUE THREAD

**Issue by saylesc** [open-dis/DISWebGateway#1]
I have an instance of this running in a Docker container. I modified the src in Configuration.java in my copy so that the ports are configurable. Maybe this is something you could add or allow me to submit a PR if it is useful to others? Thanks. 

Here's the gist:
```
String webserverPort = settings.getProperty("webserverPort");
String httpsWebserverPort= settings.getProperty("httpsWebserverPort");

// Overrides the default ports if the ENV VARS exist
if ( System.getenv("HTTP_WEBSERVER_PORT") != null )
{
   webserverPort = System.getenv("HTTP_WEBSERVER_PORT");
}
if ( System.getenv("HTTPS_WEBSERVER_PORT") != null )
{
   httpsWebserverPort = System.getenv("HTTPS_WEBSERVER_PORT");
}

HTTP_WEBSERVER_PORT = Integer.parseInt(webserverPort);
HTTPS_WEBSERVER_PORT = Integer.parseInt(httpsWebserverPort);](url)```

**Comment by mcgredonps** [comment]
I'm still getting used to git, but my understanding is that you can fork it from the original git to your own project on github, check in your changes to your project, then make the PR back to me. It seems to be easier to integrate your changes.

It's nice that you're interested in the issue. Personally I think that web-based DIS is important, and has been for years. There's lots of things on the application code side changing fast, though.

**Comment by leif81** [comment]
Fixed in #2

**Comment by mcgredonps** [comment]
Thanks. Man, the email deliveries don't show up for 24+ hours around

here. It's awful.

________________________________
